graylog2-web-interface/src/routing/Routes.tsx

```diff
@@ -19,6 +19,19 @@ import URI from 'urijs';
 
 import AppConfig from 'util/AppConfig';
 import { extendedSearchPath, viewsPath } from 'views/Constants';
+import { TimeRangeTypes } from 'views/logic/queries/Query';
+
+type RoutesRelativeTimeRange = {
+  relative: number
+};
+type RoutesAbsoluteTimeRange = {
+  from: string,
+  to: string,
```

Comment:
What about using a `number` for `from` and `to` as well? It just provides a small benefit, but this way we would ensure that we do not define any string for these attributes.

Reply:
This type is actually correct, I mixed it up with the following relative time range type: `{ type: "relative", to: number, from: number }` which is not yet defined. As discussed we can add this type when we need it.